src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -60,13 +63,14 @@ public long GetFeePerByte(StoreView snapshot)
         }
 
         [ContractMethod(0_01000000, CallFlags.AllowStates)]
-        public UInt160[] GetBlockedAccounts(StoreView snapshot)
+        public IIterator GetBlockedAccounts(StoreView snapshot)
```

Comment:
Maybe we can remove this method?

Reply:
Remove it or add `offset` and `count`, what do you prefer?

Reply:
Remove.

Reply:
Done